ansible/roles/config-idm-server/tasks/prep.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- name: Install required packages
+  package:
+    name: "{{ idm_rpms }}"
+    state: installed
+
+# Cannot use a handler here
+- name: Ensure firewalld is running
+  service:
+    name: firewalld
+    state: started
+    enabled: yes
+
+- name: Open Firewall for IdM use
+  firewalld:
+    service: "{{ item }}"
+    permanent: yes
+    state: enabled
+    immediate: yes
+  with_items:
+  - ntp
+  - http
+  - https
+  - ldap
+  - ldaps
+  - kerberos
+  - kpasswd
```

Comment:
If we install firewalld on the bastion, do we need to enable SSH too ?

Reply:
Isn't that enabled in another module?  I think this is a separation of concerns issue.